recipe/meta.yaml

```diff
@@ -41,13 +48,16 @@ requirements:
 
 test:
   commands:
-    - test -f $PREFIX/lib/libhdfs3.so
+    - test -f $PREFIX/lib/libhdfs3.so  # [linux]
```

Comment:
you might want to test here for .dylib variant on osx too?

Reply:
My first intention was to test it on `linux` and `osx`. But now we skip `osx` because it failed on it. I'll remove this constrain

Reply:
Done